cardinal/world.go

```diff
@@ -553,6 +548,43 @@ func (w *World) UseNonce(signerAddress string, nonce uint64) error {
 	return w.redisStorage.UseNonce(signerAddress, nonce)
 }
 
+func (w *World) GetDebugState() (types.EntityStateResponse, error) {
```

Comment:
See comments above re: types. Return type here should just be similar to EvaluateCQL
```
func (w *World) GetDebugState() ([]EntityStateElement, error) {
```

Reply:
Got it, will change tomorrow.